var/spack/repos/builtin/packages/akantu/package.py

```diff
@@ -65,9 +65,8 @@ def cmake_args(self):
             "-DAKANTU_HEAT_TRANSFER:BOOL=ON",
             "-DAKANTU_SOLID_MECHANICS:BOOL=ON",
             "-DAKANTU_STRUCTURAL_MECHANICS:BOOL=OFF",
-            "-DAKANTU_PARALLEL:BOOL={0}".format("ON" if spec.satisfies("+mpi") else "OFF"),
-            "-DAKANTU_PYTHON_INTERFACE:BOOL={0}".format(
-                "ON" if spec.satisfies("+python") else "OFF"
+            f"-DAKANTU_PARALLEL:BOOL={'ON' if spec.satisfies('+mpi') else 'OFF'}",
+            f"-DAKANTU_PYTHON_INTERFACE:BOOL={'ON' if spec.satisfies('+python') else 'OFF'}"
             ),
```

Comment:
```suggestion
            f"-DAKANTU_PYTHON_INTERFACE:BOOL={'ON' if spec.satisfies('+python') else 'OFF'}",
```
Sorry my previous suggestion was buggy there was a parenthesis remaining